[2021-03-02 13:31:37,701] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: titanic_pivot.download_titanic_dataset 2021-03-01T13:43:51.253838+00:00 [queued]>
[2021-03-02 13:31:37,722] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: titanic_pivot.download_titanic_dataset 2021-03-01T13:43:51.253838+00:00 [queued]>
[2021-03-02 13:31:37,722] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-03-02 13:31:37,722] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2021-03-02 13:31:37,722] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-03-02 13:31:37,729] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): download_titanic_dataset> on 2021-03-01T13:43:51.253838+00:00
[2021-03-02 13:31:37,731] {standard_task_runner.py:52} INFO - Started process 28386 to run task
[2021-03-02 13:31:37,734] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'titanic_pivot', 'download_titanic_dataset', '2021-03-01T13:43:51.253838+00:00', '--job-id', '12', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/titanicTest.py', '--cfg-path', '/tmp/tmp8vgbpcsk', '--error-file', '/tmp/tmp6p_icxbp']
[2021-03-02 13:31:37,734] {standard_task_runner.py:77} INFO - Job 12: Subtask download_titanic_dataset
[2021-03-02 13:31:37,769] {logging_mixin.py:104} INFO - Running <TaskInstance: titanic_pivot.download_titanic_dataset 2021-03-01T13:43:51.253838+00:00 [running]> on host oleg-desktop
[2021-03-02 13:31:37,837] {taskinstance.py:1255} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=titanic_pivot
AIRFLOW_CTX_TASK_ID=download_titanic_dataset
AIRFLOW_CTX_EXECUTION_DATE=2021-03-01T13:43:51.253838+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-03-01T13:43:51.253838+00:00
[2021-03-02 13:31:38,921] {python.py:118} INFO - Done. Returned value was: None
[2021-03-02 13:31:38,971] {taskinstance.py:1159} INFO - Marking task as SUCCESS. dag_id=titanic_pivot, task_id=download_titanic_dataset, execution_date=20210301T134351, start_date=20210302T103137, end_date=20210302T103138
[2021-03-02 13:31:39,004] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-03-02 13:31:39,027] {local_task_job.py:146} INFO - Task exited with return code 0
